boot/assets.go

```diff
@@ -344,6 +347,10 @@ func (o *TrustedAssetsInstallObserver) ChosenEncryptionKeys(key, saveKey secboot
 	o.saveEncryptionKey = saveKey
 }
 
+func (o *TrustedAssetsInstallObserver) SetAuxKey(key *secboot.AuxKey) {
```

Comment:
secboot.EncryptionKey is also an array type but is passed around by value - was it intentional to pass this around by pointer?